docs/guides/colibri-td-pde.ipynb

```diff
@@ -0,0 +1,439 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "73390a19",
+   "metadata": {},
+   "source": [
+    "## Get started\n",
+    "\n",
+    "Fill out the [form to request access to the QUICK-PDE function.](https://forms.office.com/e/kJzViZNVa4) Then, authenticate using your [API key](http://quantum.cloud.ibm.com/) and select the function as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "95a715d2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_ibm_catalog import QiskitFunctionsCatalog\n",
+    "\n",
+    "catalog = QiskitFunctionsCatalog(\n",
+    "    channel=\"ibm_cloud / ibm_quantum / ibm_quantum_platform\",\n",
+    "    instance=\"USER_CRN / HGP\",\n",
+    "    token=\"USER_API_KEY / IQP_API_TOKEN\",\n",
+    ")\n",
+    "\n",
+    "quick = catalog.load(\"colibritd/quick-pde\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e8837f5f",
+   "metadata": {},
+   "source": [
+    "## Examples\n",
+    "\n",
+    "To get started, try one of the following examples:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "446ac943",
+   "metadata": {},
+   "source": [
+    "### Computational Fluid Dynamics (CFD)\n",
+    "\n",
+    "When initial conditions are set to $u(0,x) = x$, the results are as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d56e1440",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# launch the simulation with initial conditions u(0,x) = a*x + b\n",
+    "job = quick.run(use_case=\"cfd\", physical_parameters={\"a\": 1.0, \"b\": 0.0})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03998691",
+   "metadata": {},
+   "source": [
+    "Check your Qiskit Function workload's [status](/docs/guides/functions#check-job-status) or return [results](/docs/guides/functions#retrieve-results) as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "856fe992",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(job.status())\n",
+    "solution = job.result()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c42aba9b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "_ = plt.figure()\n",
+    "ax = plt.axes(projection=\"3d\")\n",
+    "\n",
+    "# plot the solution using the 3d plotting capabilities of pyplot\n",
+    "t, x = np.meshgrid(solution[\"samples\"][\"t\"], solution[\"samples\"][\"x\"])\n",
+    "ax.plot_surface(\n",
+    "    t,\n",
+    "    x,\n",
+    "    solution[\"functions\"][\"u\"],\n",
+    "    edgecolor=\"royalblue\",\n",
+    "    lw=0.25,\n",
+    "    rstride=26,\n",
+    "    cstride=26,\n",
+    "    alpha=0.3,\n",
+    ")\n",
+    "ax.scatter(t * 5, x, solution, marker=\".\")\n",
```

Comment:
Changed `t_unit` to `t` because it raised an undefined error - is that correct @Henri-ColibrITD?